Docker/test/selenium/helpers/default-functions.js

```diff
@@ -14,6 +15,29 @@ const defaultFunctions = function () {
 		return axios.get( url );
 	} );
 
+	/**
+	 * Make a get request to get full request response
+	 */
+	browser.addCommand( 'deleteProperty', function async( claimGuid ) {
```

Comment:
This seems to delete a statement rather than the actual property.

Reply:
done